docs/resources.md

```diff
@@ -13,5 +13,5 @@ Name | Description
 -----| -----------
 [jgrade](https://github.com/tkutche1/jgrade) | A Java helper library for building autograders for Gradescope
 [UCSB Gradescope Tools](https://ucsb-gradescope-tools.github.io/) | Tools and examples for C++ and Python autograders, pulling from GitHub, etc
-[GradescopeGrader4Autolab](https://github.com/xyzisinus/GradescopeGrader4Autolab) | Tools to help port Autolab autograders to Gradescope
+[GradescopeGrader4Autolab](https://github.com/xyzisinus/GradescopeGrader4Autolab) | A bridging grader to help port Autolab autograders to Gradescope
```

Comment:
Hi @xyzisinus, thanks for your contribution! Is "Bridging grader" some Autolab-specific terminology? I don't know if that term would make sense to other people. Perhaps something like this would work?

```suggestion
[GradescopeGrader4Autolab](https://github.com/xyzisinus/GradescopeGrader4Autolab) | A wrapper autograder to help port Autolab autograders to Gradescope
```

I haven't really seen the term "bridging" used in this context so I feel like it might be confusing to users.